Makefile

```diff
@@ -9,12 +9,12 @@ SUBDIRS := src src/hmm src/thirdparty src/common src/alignment
 
 #Basic flags every build needs
 LIBS=-lz
-CXXFLAGS = -g -O3
+CXXFLAGS ?= -g -O3
 CXXFLAGS += -std=c++11 -fopenmp
-CFLAGS=-O3
-CXX=g++
-CC=gcc
-HDF5=install
+CFLAGS ?= -O3
+CXX ?= g++
+CC ?= gcc
+HDF5=install # change to any value to disable compilation of bundled HDF5 code
```

Comment:
This inline comment breaks the auto-installation of HDF5, please put the comment on the preceding line